src/components/swap/SwapModalHeader.tsx

```diff
@@ -99,20 +107,29 @@ export default function SwapModalHeader({
         </SwapShowAcceptChanges>
       ) : null}
       <AutoColumn justify="flex-start" gap="sm" style={{ padding: '12px 0 0 0px' }}>
-        {trade.tradeType === TradeType.EXACT_INPUT ? (
+        {isRoutedViaStableSwap || trade.tradeType === TradeType.EXACT_INPUT ? (
           <TYPE.italic textAlign="left" style={{ width: '100%' }}>
             {t('swap.outputEstimated')}
-            <b>
-              {slippageAdjustedAmounts[Field.OUTPUT]?.toSignificant(6)} {trade.outputAmount.currency.symbol}
-            </b>
+            {
+              <b>
```

Comment:
[Should this be <strong>?](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#b_vs._strong)